lib/Graph/Graph.cpp

```diff
@@ -2420,14 +2430,22 @@ void Function::verify() const {
         continue;
       }
       auto varToFirstDef = placeholderWrittenTo.find(ph);
-      if (varToFirstDef != placeholderWrittenTo.end()) {
-        llvm::errs() << "Placeholder " << ph->getDebugDesc() << '\n';
-        llvm::errs() << "has more than one write:\n";
-        llvm::errs() << N.getDebugDesc() << '\n';
-        llvm::errs() << varToFirstDef->second->getDebugDesc() << '\n';
+      bool writtenOnce = expectCompareTrue(
+          "Placeholder has more than one write",
+          varToFirstDef == placeholderWrittenTo.end(), true, ph);
+      if (!writtenOnce) {
+        isValid = false;
+        std::string storage;
+        llvm::raw_string_ostream msg(storage);
+
+        msg << "Placeholder " << ph->getDebugDesc() << '\n';
+        msg << "has more than one write:\n";
```

Comment:
nit: I know this is copied over, but I find this error message a little confusing, since we only print one of the writers. Maybe this could say something like `"has more than one write; second writer found:"` ?

Reply:
Done.